src/Synapse/Synapse/help/Set-AzSynapseIntegrationRuntime.md

```diff
@@ -0,0 +1,649 @@
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InputObject
+The integration runtime object.
+
+```yaml
+Type: Microsoft.Azure.Commands.Synapse.Models.PSIntegrationRuntime
+Parameter Sets: SetByIntegrationRuntimeObject, SetByLinkedIntegrationRuntimeObject
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -LicenseType
+The license type that you want to select for the SSIS IR.
+There are two types: LicenseIncluded or BasePrice.
+If you are qualified for the Azure Hybrid Use Benefit (AHUB) pricing, please select BasePrice.
+If not, please select LicenseIncluded.
+
+```yaml
+Type: System.String
+Parameter Sets: SetByIntegrationRuntimeName, SetByParentObject, SetByResourceId, SetByIntegrationRuntimeObject
+Aliases:
+Accepted values: LicenseIncluded, BasePrice
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Location
+The integration runtime description.
+
+```yaml
+Type: System.String
+Parameter Sets: SetByIntegrationRuntimeName, SetByParentObject, SetByResourceId, SetByIntegrationRuntimeObject
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -MaxParallelExecutionsPerNode
+Maximum parallel execution count per node for a managed dedicated integration runtime.
+
+```yaml
+Type: System.Nullable`1[System.Int32]
+Parameter Sets: SetByIntegrationRuntimeName, SetByParentObject, SetByResourceId, SetByIntegrationRuntimeObject
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Name
+The integration runtime name.
+
+```yaml
+Type: System.String
+Parameter Sets: SetByIntegrationRuntimeName, SetByLinkedIntegrationRuntimeName, SetByParentObject, SetByLinkedIntegrationRuntimeParentObject
+Aliases: IntegrationRuntimeName
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -NodeCount
+Target nodes count of the integration runtime.
+
+```yaml
+Type: System.Nullable`1[System.Int32]
+Parameter Sets: SetByIntegrationRuntimeName, SetByParentObject, SetByResourceId, SetByIntegrationRuntimeObject
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -NodeSize
+The integration runtime node size.
+
+```yaml
+Type: System.String
+Parameter Sets: SetByIntegrationRuntimeName, SetByParentObject, SetByResourceId, SetByIntegrationRuntimeObject
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PublicIPs
```

Comment:
Plural parameter name is against PowerShell design guideline. Please change it to `PublicIP`. You may keep `PublicIPs` as an alias to consist with ADF